layouts/community/ortho_5x15/wanleg/keymap.c

```diff
@@ -1,26 +1,71 @@
 #include QMK_KEYBOARD_H
 #include "wanleg.h"
 
-#define _________________BLANK_75__________________ _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
 #define _________________BLANK_50__________________ _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
-#define _________________Num_Row_75________________ KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_NLCK
+
+#if defined(PADC)
+#define LAYOUT_75_base( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0a, K0b, K0c, K0d, K0e, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1a, K1b, K1c, K1d, K1e, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2a, K2b, K2c, K2d, K2e, \
+	K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3a, K3b, K3c, K3d, K3e  \
+  ) \
+  LAYOUT_wrapper( \
+     KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_NLCK,  QWERTY, GHERKIN, FUNCTION, \
+     K00, K01, K02, K03, K04, K05, 		K0c,  K0d,  K0e,		K06, K07, K08, K09, K0a, K0b, \
+     K10, K11, K12, K13, K14, K15, 		K1c,  K1d,  K1e,		K16, K17, K18, K19, K1a, K1b, \
+     K20, K21, K22, K23, K24, K25, 		K2c,  K2d,  K2e,		K26, K27, K28, K29, K2a, K2b, \
+     K30, K31, K32, K33, K34, K35, 		K3c,  K3d,  K3e,	    K36, K37, K38, K39, K3a, K3b  \
+  )
+
+#elif defined(PADL)
+#define LAYOUT_75_base( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0a, K0b, K0c, K0d, K0e, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1a, K1b, K1c, K1d, K1e, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2a, K2b, K2c, K2d, K2e, \
+	K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3a, K3b, K3c, K3d, K3e  \
+  ) \
+  LAYOUT_wrapper( \
+     KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_NLCK,  QWERTY, GHERKIN, FUNCTION, \
+     K0c,  K0d,  K0e,		K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0a, K0b, \
+     K1c,  K1d,  K1e, 		K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1a, K1b, \
+     K2c,  K2d,  K2e, 		K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2a, K2b, \
+     K3c,  K3d,  K3e,  		K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3a, K3b 	\
+  )
+  
+#else
+#define LAYOUT_75_base( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0a, K0b, K0c, K0d, K0e, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1a, K1b, K1c, K1d, K1e, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2a, K2b, K2c, K2d, K2e, \
+	K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3a, K3b, K3c, K3d, K3e  \
+  ) \
+  LAYOUT_wrapper( \
```

Comment:
```suggestion
   LAYOUT_ortho_5x15_wrapper( \
```

Reply:
ha! that's it! thank you so much!

Reply:
welcome. :)

if you ever have questions about something like this, don't hesitate to ping me. :)